packages/design-editor/src/shell/shell.css

```
@@ -0,0 +1,83 @@
+ * our bundle, but the shell document and a scene frame share an origin and the
+ * frame's stylesheet is the consumer's. An unprefixed `--background` would be a
+ * coin flip.
+ */
+@import 'tailwindcss';
+
+/*
+ * Scanning is scoped to the shell's own tree. Without this Tailwind would look at
+ * the package root and pull candidates out of `src/plugin/**` — Node code whose
+ * string literals are file paths and CSS property names, not classes.
+ */
+@source './';
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)